app/renderer/js/utils/logger-util.js

```diff
@@ -5,14 +5,29 @@ const { initSetUp } = require('./default-util');
 const { sentryInit, captureException } = require('./sentry-util');
 
 initSetUp();
-sentryInit();
+
```

Comment:
@vsvipul did you check up with @abhigyank for this?

Reply:
Yes i did, this was done because sentryInit() needs to be done separately for both processes. I've handled that here.